pom.xml

```diff
@@ -140,7 +162,12 @@
                 <version>${jaxb-runtime.version}</version>
                 <scope>runtime</scope>
             </dependency>
-
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>farao-crac-impl</artifactId>
+                <scope>runtime</scope>
+                <version>${farao.core.version}</version>
+            </dependency>
```

Comment:
Same remark: this should not be necessary as it is already included in farao-parent-bom

Reply:
yes